### ISSUE TITLE
minor doc glitch?

### ISSUE THREAD

**Issue by alto777** [david-c14/SubmarineFree#10]
C-14!

I make no sense of the "normalling" section of the page on Pulse Generators, I wonder if this is accidental cut and paste?

Also, I was combing through your other logic elements and wondered if you have a plan to offer JK flip flops. I drew an idea for one that takes about 30mm square, has all J, K, CLK, SET and CLEAR inputs and Q, /Q outputs. Eight look a bit cramped on a 12U panel, no room for  Submarine tho. The SET and CLEAR could be lost, mebbe just one global CLEAR (reset).

I am reminded of a logic laboratory I used some time ago, never mind precisely how long. It had four JKs, some debounced switches and a clock device I think and a smattering of gates and lamps with which little machines could be built.

I am enjoying watching various ways people are exploiting the Rack architecture.

a7

**Comment by david-c14** [comment]
Yes, I have been thinking about other logic gates and that includes sr and jk flip flops. I don't know where they sit on my huge wish list. My highest priorities tend to be listed in these issues with the wishlist tag.

**Comment by david-c14** [comment]
It was a cut and paste error on the pulse gen manual page. Thanks.
